step1_SMS-PM/stash.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/python
```

Comment:
This looks like being a file you are locally using and shouldn't be part of the repository either.